nixos/modules/services/web-servers/nginx/tailscale-auth.nix

```
@@ -52,66 +46,14 @@ in
 
   config = mkIf cfg.enable {
     services.tailscale.enable = true;
+    services.tailscaleAuth.enable = true;
```

```suggestion
    services.tailscaleAuth.enable = true;
```
Nit: Technically unnecessary as `tailscaleAuth` already enables tailscale.

---

Thanks!